package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-document-generator",
-  "version": "0.12.1",
+  "version": "0.12.2",
```

Comment:
can probably go to 0.13.0 as its not a bug fix and it is a non breaking feature change (https://semver.org/)